FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -35,12 +35,14 @@
  * with another MQTT library. This demo requires using the AWS IoT broker as
  * Device Defender is an AWS service.
  *
- * This demo connects to the AWS IoT broker and subscribes to the device
- * defender topics. It then collects metrics for the open ports and sockets on
- * the device using FreeRTOS+TCP, and generates a device defender report. The
+ * This demo subscribes to the device defender topics. It then collects metrics
+ * for the open ports and sockets on the device using FreeRTOS+TCP. Additonally
+ * the stack high water mark and task ids are collected for custom metrics.
+ * These metrics are used to generate a device defender report. The
  * report is then published, and the demo waits for a response from the device
- * defender service. Upon receiving the response or timing out, the demo
- * finishes.
+ * defender service. Upon receiving an accepted response, the demo finishes.
+ * If the demo recieves a rejected response or times out, the demo repeats up to
```

Comment:
spelling: _receives_.  It looks like the spell check approval workflow is not running on this repository, please consider adding it.